### PR TITLE
General CS fixes

### PR DESCRIPTION
**Pull request by Zales0123**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets |
| License       | MIT
| Doc PR        |

After merging #2978 @michalmarcinkowski rightly noticed some CS issues that could and should be fixed. Here they are. I have also deleted some "return" statements in other ``PaymentMethod`` functions (which may cause some potential BC breaks) and some irritating unused use statements introduced by Scrutinizer.

### CODE REVIEW

**Comment by michalmarcinkowski** [comment]
:+1: 

Please add this BC Break (remove fluent interfaces from PaymentMethod) to the changelog (simply open PR to my branch https://github.com/Sylius/Sylius/pull/2990).

File: src/Sylius/Component/Payment/Model/PaymentMethod.php [hunk 1]
```diff
@@ -144,8 +142,6 @@ public function getName()
     public function setName($name)
     {
         $this->name = $name;
-
-        return $this;
```
Comment (Shine-neko) [inline src/Sylius/Component/Payment/Model/PaymentMethod.php]: why ?

Comment (Zales0123) [inline src/Sylius/Component/Payment/Model/PaymentMethod.php]: As @michalmarcinkowski wrote in my previous PR, we no longer do fluent interfaces in Sylius.

Comment (michalmarcinkowski) [inline src/Sylius/Component/Payment/Model/PaymentMethod.php]: @Shine-neko http://ocramius.github.io/blog/fluent-interfaces-are-evil/

**Pull request merged**